src/test/identity-manager/WorldIDIdentityManagerTest.sol

```diff
@@ -105,14 +115,18 @@ contract WorldIDIdentityManagerTest is WorldIDTest {
     /// @notice Initialises a new identity manager using the provided information.
     /// @dev It is initialised in the globals.
     ///
+    /// @param actualTreeDepth The tree depth to use.
     /// @param actualPreRoot The pre-root to use.
-    /// @param actualVerifier The verifier instance to use.
+    /// @param actualTreeVerifier The tree verifier instance to use.
+    /// @param actualSemaphoreVerifier The Semaphore verifier instance to use.
     /// @param enableStateBridge Whether or not the new identity manager should have the state
     ///        bridge enabled.
     /// @param actualStateBridgeProxy The address of the state bridge.
     function makeNewIdentityManager(
+        uint8 actualTreeDepth,
         uint256 actualPreRoot,
-        ITreeVerifier actualVerifier,
+        ITreeVerifier actualTreeVerifier,
+        ISemaphoreVerifier actualSemaphoreVerifier,
```

Comment:
Unless you're actually using the ability to pass this here, just construct it inside the `makeNewIdentityManager` function. Same for `actualTreeDepth`.

Reply:
I've added tests that use it.